### ISSUE TITLE
Jakarta.ee Website Refresh

### ISSUE THREAD

**Issue by chrisguindon** [jakartaee/jakarta.ee#1236]
I am creating this issue to allow us to have high-level conversations around Jakarta.ee Website Refresh project: https://github.com/jakartaee/jakarta.ee/projects/1

Before we start, it's important to understand that the look and feel of the Jakarta.ee re-design was already discussed and approved: https://github.com/jakartaee/jakarta.ee/issues/1070. 

The focus of this project is now on the implementation of this new l&f: https://projects.invisionapp.com/share/MV1064CX7GC6#/screens/444513792

We are planning a phased delivery approach with this project. 

## Q3 - Template + Homepage

_Due by September 30, 2021_

In Q3, our focus will be on updating core components that we plan on using on every page. This includes components like our header, footer, main menu, and our Jumbotron. We will also include the implementation of our new homepage with our Q3 delivery:
https://github.com/jakartaee/jakarta.ee/milestone/1

- [x] https://github.com/jakartaee/jakarta.ee/issues/1133
- [x] https://github.com/jakartaee/jakarta.ee/issues/1134
- [x] https://github.com/jakartaee/jakarta.ee/issues/1135
- [x] https://github.com/jakartaee/jakarta.ee/issues/1136
- [x] https://github.com/jakartaee/jakarta.ee/issues/1137
- [x] https://github.com/jakartaee/jakarta.ee/issues/1138
- [x] https://github.com/jakartaee/jakarta.ee/issues/1139
- [x] https://github.com/jakartaee/jakarta.ee/issues/1140
- [x] https://github.com/jakartaee/jakarta.ee/issues/1162

### Q3 - Nice to have (Enhancement) 

- [x] https://github.com/jakartaee/jakarta.ee/issues/1157

## Q4 - Individual pages

_Due by December 31, 2021_

In Q4, we will work on implementing all the secondary pages of the site:
https://github.com/jakartaee/jakarta.ee/milestone/3

- [x] https://github.com/jakartaee/jakarta.ee/issues/1141
- [x] https://github.com/jakartaee/jakarta.ee/issues/1142
- [x] https://github.com/jakartaee/jakarta.ee/issues/1143
- [x] https://github.com/jakartaee/jakarta.ee/issues/1144
- [x] https://github.com/jakartaee/jakarta.ee/issues/1145
- [x] https://github.com/jakartaee/jakarta.ee/issues/1146
- [x] https://github.com/jakartaee/jakarta.ee/issues/1147
- [x] https://github.com/jakartaee/jakarta.ee/issues/1148
- [x] https://github.com/jakartaee/jakarta.ee/issues/1149
- [x] https://github.com/jakartaee/jakarta.ee/issues/1150
- [x] https://github.com/jakartaee/jakarta.ee/issues/1151
- [x] https://github.com/jakartaee/jakarta.ee/issues/1152
- [x] https://github.com/jakartaee/jakarta.ee/issues/1153
- [x] https://github.com/jakartaee/jakarta.ee/issues/1154
- [x] https://github.com/jakartaee/jakarta.ee/issues/1166
- [x] https://github.com/jakartaee/jakarta.ee/issues/1167
- [x] https://github.com/jakartaee/jakarta.ee/issues/1289
- [x] https://github.com/jakartaee/jakarta.ee/issues/1290
- [x] https://github.com/jakartaee/jakarta.ee/issues/1324

### Q4 - Nice to have (Enhancements / New features) 

- [x] https://github.com/jakartaee/jakarta.ee/issues/1348
- [x] https://github.com/jakartaee/jakarta.ee/issues/1346

For Phase 1 (Q3), the work will likely be delivered by the end of the quarter. However, for phase 2 (q4), we plan on rolling out individual page updates as they are ready.

### Merge Compatible Products
- [x] https://github.com/jakartaee/jakarta.ee/issues/1166
- [x] https://github.com/jakartaee/jakarta.ee/issues/1167
- [x] https://github.com/jakartaee/jakarta.ee/issues/1174

### Implement widgets

- [x] https://github.com/jakartaee/jakarta.ee/issues/1173
- [x] https://github.com/jakartaee/jakarta.ee/issues/1161
- [x] https://github.com/jakartaee/jakarta.ee/issues/1160

### Translate new content to second language

- [x] https://github.com/jakartaee/jakarta.ee/issues/1338

### Update to the News section

- [x] https://github.com/jakartaee/jakarta.ee/issues/1321

### Update CSS for JakartaBlogs

- [x] https://github.com/jakartaee/jakartablogs.ee/issues/106

### Blogs

- [x] https://github.com/jakartaee/jakarta.ee/issues/1371

**Comment by chrisguindon** [comment]
The current state of this project can be seen here: https://redesign--jakartaee.netlify.app/

**Comment by chrisguindon** [comment]
@ericpoirier @shabnammayel,

This is what I am thinking as a timeline for September:

- September 13-17: We finish reviewing the site and agree on the final changes that we need to make https://deploy-preview-1256--jakartaee.netlify.app/
- September 17-22: Allocated for community feedback and testing
- September 23-29: Finalize last changes
- September 30: Go live with phase 1

**Comment by ericpoirier** [comment]
> @ericpoirier @shabnammayel,
> 
> This is what I am thinking as a timeline for September:
> 
> * September 13-17: We finish reviewing the site and agree on the final changes that we need to make https://deploy-preview-1256--jakartaee.netlify.app/
> * September 17-22: Allocated for community feedback and testing
> * September 23-29: Finalize last changes
> * September 30: Go live with phase 1

+1

**Comment by ericpoirier** [comment]
I made changes based on the changes we talked about yesterday:
https://deploy-preview-1256--jakartaee.netlify.app/

The changes that I've made are listed here:
https://docs.google.com/document/d/1sC0-N4sj0WdP-vUpC3v7tyspTnS8DmwlKy7QmYLf_IU/edit#

Let me know if there's anything else.

Also I noticed that the navigation from the following document is different from what we currently have on staging:
https://docs.google.com/spreadsheets/d/1oB21kaN-25slUfZm5Mq2IgCgx1ky-HYxu71D5FTXRWQ/edit?pli=1#gid=1312835957

Should I update the staging navigation based on this spreadsheet?

Thanks.

**Comment by chrisguindon** [comment]
> Should I update the staging navigation based on this spreadsheet?

@shabnammayel is planning to give us an updated sitemap-navigation tomorrow. Let's wait for that update before we do anything.

**Comment by ericpoirier** [comment]
> > Should I update the staging navigation based on this spreadsheet?
> 
> @shabnammayel is planning to give us an updated sitemap-navigation tomorrow. Let's wait for that update before we do anything.

Ok will do.

**Comment by shabnammayel** [comment]
@ericpoirier I have reviewed the changes you made and added some comments here in your google doc: https://docs.google.com/document/d/1sC0-N4sj0WdP-vUpC3v7tyspTnS8DmwlKy7QmYLf_IU/edit#
Also shared the changes with @ivargrimstad and @TanjaObradovic to get their feedback

**Comment by ericpoirier** [comment]
> @ericpoirier I have reviewed the changes you made and added some comments here in your google doc: https://docs.google.com/document/d/1sC0-N4sj0WdP-vUpC3v7tyspTnS8DmwlKy7QmYLf_IU/edit#
> Also shared the changes with @ivargrimstad and @TanjaObradovic to get their feedback

Thanks!

I'm thinking that maybe I should wait until the sitemap-navigation is entirely done before making more changes to the menu. This way I would make changes only once and there will be less back and forth. What do you think?

**Comment by shabnammayel** [comment]
I don't think I have more suggested changes than what we discussed yesterday and what I added to the document today. I went through the current website to make sure we are including all of the links in the new design.
so that's it from my side. I asked Tanja and Ivar to review and provide feedback too. thanks Eric.

**Comment by edbratt** [comment]
FWIW, I looked quickly at the deploy preview and the site update looks really nice! It might be useful to update the Marketing Committee with a note about this progress and the proposed roll-out. My thanks to all of you for this work!

**Comment by shabnammayel** [comment]
Thanks for going through the previews @edbratt. I am planning to provide an overview of the updates on our Marketing Committee Call tomorrow and based on the timeline that Chris provided, mention that the updates are open for community feedback and testing from Sep 17-24.

**Comment by chrisguindon** [comment]
@shabnammayel What would be the best way to announce to the community that we are looking for some feedback? 

Is there a mailing list we should use? I am thinking we should point people to this issue for feedback on Friday once we are done with our internal changes.

**Comment by ericpoirier** [comment]
@shabnammayel I made the changes to the main menu:

https://deploy-preview-1256--jakartaee.netlify.app/

Let me know if i missed anything.

Thanks.

**Comment by ericpoirier** [comment]
@shabnammayel I did the last changes for the main menu based on the discussion we had yesterday afternoon:

https://deploy-preview-1256--jakartaee.netlify.app/

Let me know if there's anything else.

**Comment by shabnammayel** [comment]
Thanks @ericpoirier looks great! News is missing from the "About" dropdown. It should be moved from resource dropdown to "About" dropdown at the bottom as indicated here: https://docs.google.com/spreadsheets/d/1oB21kaN-25slUfZm5Mq2IgCgx1ky-HYxu71D5FTXRWQ/edit?pli=1#gid=1312835957 
Thanks

**Comment by ericpoirier** [comment]
> Thanks @ericpoirier looks great! News is missing from the "About" dropdown. It should be moved from resource dropdown to "About" dropdown at the bottom as indicated here: https://docs.google.com/spreadsheets/d/1oB21kaN-25slUfZm5Mq2IgCgx1ky-HYxu71D5FTXRWQ/edit?pli=1#gid=1312835957
> Thanks

I thought this one was going in resources but I moved it.

https://deploy-preview-1256--jakartaee.netlify.app/

Let me know if I missed anything else.

**Comment by shabnammayel** [comment]
Thanks @ericpoirier , Based on the discussions on the Marketing Committee call today, members will work on updating the testimonials and limiting them to 250 characters. and also will provide the missing images of the quoters. I think your recommended image size is 20kb~ and around 300x300 pixels, right? The deadline to submit the revised quotes under this issue is EOD Sep 24 so that you can have the next week to finalize the changes.

**Comment by chrisguindon** [comment]
@shabnammayel How do you recommend we reach out to the community about this on Friday? Is there a mailing list we should use to point people to this issue?

**Comment by shabnammayel** [comment]
re testimonials MKT committee members will be able to provide the updated testimonials. but for the general community feedback on the updates we can inform through the community mailing list.

**Comment by shabnammayel** [comment]
@ericpoirier 2 more edits :) 1. The link under Get Involved to join the working group in the box in homepage is going to the charter. it should lead to the membership page. 2. Please change the title of the Videos on homepage to " Jakarta EE on YouTube"and change the blurb to "Jakarta EE Youtube channel is a platform that features Jakarta EE related videos aiming to educate the new and existing community members about open source cloud native and microservices technologies" thanks

**Comment by ericpoirier** [comment]
> @ericpoirier 2 more edits :) 1. The link under Get Involved to join the working group in the box in homepage is going to the charter. it should lead to the membership page. 2. Please change the title of the Videos on homepage to " Jakarta EE on YouTube"and change the blurb to "Jakarta EE Youtube channel is a platform that features Jakarta EE related videos aiming to educate the new and existing community members about open source cloud native and microservices technologies" thanks

I made those changes to the following staging page:
https://deploy-preview-1262--jakartaee.netlify.app/

Let me know if there's anything else.

Thanks.

**Comment by cesarhernandezgt** [comment]
Hi @ericpoirier 
I know the current scope is on the home page, but it seems the changes in the theme made 9.1 release page overlapping a bit in the banner around class `jakartaee-9-flex-grow-container vertical-align flex-wrap`

Screenshot from a 1920x1080 resolution:
![image](https://user-images.githubusercontent.com/1454523/133817876-0f7d3a55-a9a0-4bfb-8bf2-efd181e011ca.png)

**Comment by ericpoirier** [comment]
> Hi @ericpoirier
> I know the current scope is on the home page, but it seems the changes in the theme made 9.1 release page overlapping a bit in the banner around class `jakartaee-9-flex-grow-container vertical-align flex-wrap`
> 
> Screenshot from a 1920x1080 resolution:
> ![image](https://user-images.githubusercontent.com/1454523/133817876-0f7d3a55-a9a0-4bfb-8bf2-efd181e011ca.png)

Thanks for reporting this issue, I'll get that fixed.

Must be a regression due to update in the CSS.

**Comment by trulymobilenwp** [comment]
Here is a suggested version closer to the 250 char count.
“IBM helped create Java EE and guide it through its teenage years and I’m delighted to see the progress made under Jakarta EE. I look forward to Jakarta EE and MicroProfile becoming the open standard for cloud-native innovation and the technology we need for Java in the cloud.”
– Ian Robinson, Distinguished Engineer, CTO IBM App Platform

**Comment by shabnammayel** [comment]
Here is the updated quote from Oracle: 
"Oracle remains committed to the success of Jakarta EE and is pleased with its progress and community support. We are contributing to the projects and the committees and are leveraging the technologies in Oracle Enterprise Cloud Native Java products."
- Tom Snyder, VP of Engineering, Oracle Enterprise Cloud Native Java

**Comment by shabnammayel** [comment]
@ericpoirier we would like to create a similar page to the resource page here https://projects.invisionapp.com/share/AX10M9QL74RN#/screens/448585119 for the "Community" and "Specification" dropdown in the navigation bar so that all the link would land and be anchored to this page first instead of going to the third party website. The objective is to keep the traffic within the Jakarta EE website before driving it to an external page. The boxes content should be the list that we hav on the "Community" and "Specification" dropdown for each of the pages. Please let me know if you need further information to get this going. thanks

**Comment by chrisguindon** [comment]
@ericpoirier We need to add the implementation of the blog section for Q4 that will allow staff and community to submit blog post via a PR. The look and feel should be based off /news.

Can you create a bug, if we don't already have one for that section and add it to list please?

However, it's not a must to implement the automation (https://github.com/jakartaee/jakarta.ee/issues/1037) that will be required to import posts from jakartablogs.ee in Q4.

If we have time this quarter, we should look into that but I don't consider the automation to be part of the deliverables for the Q4 redesign phase.

**Comment by ericpoirier** [comment]
> Here is the updated quote from Oracle: "Oracle remains committed to the success of Jakarta EE and is pleased with its progress and community support. We are contributing to the projects and the committees and are leveraging the technologies in Oracle Enterprise Cloud Native Java products."
> 
> * Tom Snyder, VP of Engineering, Oracle Enterprise Cloud Native Java

Hi @shabnammayel & @trulymobilenwp, 

Sorry for the delay, can you review the updated testimonials here:

https://deploy-preview-1284--jakartaee.netlify.app/

Thanks.

**Comment by shabnammayel** [comment]
Both quotes look good to me @ericpoirier! Thanks

**Comment by shabnammayel** [comment]
Actually for consistency, can you please remove the hyphen on Cloud-native in IBM's quote?  thanks

**Comment by ericpoirier** [comment]
> Actually for consistency, can you please remove the hyphen on Cloud-native in IBM's quote? thanks

I updated the typo in all testimonials.

Please review:
https://deploy-preview-1295--jakartaee.netlify.app/

Let me know if there's anything else.

**Comment by shabnammayel** [comment]
Looks good! Thanks @ericpoirier

**Comment by shabnammayel** [comment]
@ericpoirier the Releases button in the navigation dropdown and also in the homepage under the "Get Started" section goes to Jakarta EE 9 release page. Please change them both to go to Jakarta EE 9.1 release page: https://jakarta.ee/release/9.1/. Thanks

**Comment by ericpoirier** [comment]
> @ericpoirier the Releases button in the navigation dropdown and also in the homepage under the "Get Started" section goes to Jakarta EE 9 release page. Please change them both to go to Jakarta EE 9.1 release page: https://jakarta.ee/release/9.1/. Thanks

I updated the link here:
https://deploy-preview-1303--jakartaee.netlify.app/

I'll wait for your +1 before merging.

**Comment by shabnammayel** [comment]
+1 thanks Eric!

**Comment by ericpoirier** [comment]
> +1 thanks Eric!

Merged

**Comment by chrisguindon** [comment]
@ericpoirier Can we close this issue?

**Comment by ericpoirier** [comment]
@chrisguindon I think we are done here. Just closed the remaining task that I think is also done.
